torch/fx/experimental/proxy_tensor.py

```diff
@@ -243,17 +248,17 @@ def can_handle_tensor(x):
     if not pytree.tree_all_only(torch.Tensor, can_handle_tensor, (args, kwargs)):
         return NotImplemented
 
-    if func in CURRENT_DECOMPOSITION_TABLE:
+    if not external_call:
+        if func in CURRENT_DECOMPOSITION_TABLE:
```

Comment:
Not sure if `external_func` can be in `CURRENT_DECOMPOSITION_TABLE`. Answer: no, as the decomp table is to decompose complex pytorch operators into simpler ones. Probably used in PrimTorch?